src/Reporter/AbstractBaseReporter.php

```diff
@@ -102,7 +102,13 @@ public function __construct(
      */
     public function color($key, $text)
     {
-        if (!$this->configuration->areColorsEnabled() || !$this->hasColorSupport()) {
```

Comment:
Just tried to follow PSR at my best, for my own projects I usually do it like this.

```PHP
$cond = $veryLongStatement1;
$cond = $cond && $veryLongStatement2;
$cond = $cond && $veryLongStatement3;

if ($cond) {
  // ...
}
```

But I'll go with your way this time for the sake of consistency.